### ISSUE TITLE
amp-carousel 0.2 responsive image bug

### ISSUE THREAD

**Issue by LNHarders** [ampproject/amphtml#24627]
## What's the issue?

When using layout="responsive" with amp-carousel 0.2 the images only get smaller in their width. The height stays the same which makes them look distorted.

## How do we reproduce the issue?

https://playground.amp.dev/?url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fexamples%2Fintroduction%2Fhello_world_email&format=email&_gl=1*xa1x0p*_ga*YW1wLU8tZGRBdk00aGdzZ1BwR3BuMXRDeEE.

1. open the link
2. change "https://cdn.ampproject.org/v0/amp-carousel-0.1.js" to "https://cdn.ampproject.org/v0/amp-carousel-0.2.js"

## What browsers are affected?

Seen in Safari (version 12.1.2) and Chrome (version 77.0.3865.75) on Mac OS

## Which AMP version is affected?

Latest version (1909141409590)

**Comment by sparhami** [comment]
This is due to `object-fit: contain` being missing for the imgs. This is caused by a bad selector in `amp-carousel` 0.2. #24324 fixes the issue, but I haven't had a chance to submit it yet. Will look into submitting it.
